src/xdocs/property_types.xml

```diff
@@ -908,56 +927,52 @@
       </div>
     </section>
 
-    <section name="JavadocContentLocationOption">
+    <section name="Uri">
       <p>
-        This type represents policy on placement of the Javadoc content.
+        This type represents a URI. The string representation is parsed using a custom
+        routine to analyze what type of URI the string is.
+        It can be a URL, regular file, or a resource path. It will try loading the path as
+        a URL first, then as a file that must exist, and then finally as a resource on the
+        classpath.
+      </p>
+    </section>
+
+    <section name="WrapOption">
+      <p>
+        This type represents the policy for wrapping lines.
         The following table describes the valid options:
       </p>
 
       <div class="wrapper">
-        <table summary="JavadocContentLocationOption options">
+        <table summary="wrap options">
           <tr>
             <td>Option</td>
             <td>Definition</td>
           </tr>
-
           <tr>
-            <td><code>FIRST_LINE</code></td>
+            <td><code>nl</code></td>
             <td>
-              Represents the policy for Javadoc content starts from the same line
-              as <code>/**</code>.
-              Example:
+              The token must be on a new line. For example:
               <div class="wrapper">
                 <pre>
-            &#47;** Summary text.
-              * More details.
-              *&#47;
-            public void method();
+    someVariable = aBigVariableNameToMakeThings + "this may work"
+                   + lookVeryInteresting;
                 </pre>
               </div>
-              This style is also known as "scala" style.
             </td>
           </tr>
-
           <tr>
-            <td><code>SECOND_LINE</code></td>
+            <td><code>eol</code></td>
             <td>
-              Represents the policy for Javadoc content starts from the next line
-              after <code>/**</code>.
-              Example:
+              The token must be at the end of the line. For example:
               <div class="wrapper">
                 <pre>
-              &#47;**
-              * Summary text.
-              * More details.
-              *&#47;
-              public void method();
+    someVariable = aBigVariableNameToMakeThings + "this may work" +
+                   lookVeryInteresting;
```

Comment:
I would suggest to move these sorting changes to new PR to not mix it with renaming changes. Also, please add screenshot or list how new order looks like (to make sure it is sorted properly, it is not clear from diff)

Reply:
Okay, will do it asap.

Reply:
@strkkk Done: https://github.com/checkstyle/checkstyle/pull/8400